openapi/index.yaml

```diff
@@ -66,3 +66,85 @@ definitions:
       - statusQueryGetUri
       - sendEventPostUri
       - terminatePostUri
+  ConsultazioneSogliaIndicatoreInput:
+    type: object
+    properties:
+      CodiceFiscale:
+        type: string
+        pattern: ([A-Za-z]{6}\\d{2}[A-Za-z]\\d{2}[A-Za-z][A-Za-z0-9]{3}[A-Za-z])|(\\d{11} {0,5})
+      CodiceSoglia:
+        type: string
+        pattern: ([A-Za-z]{4}[0-9]{2})
+      FornituraNucleo:
+        type: string
+        enum:
+          - SI
+          - "NO"
+      DataValidita:
+        type: string
+        pattern: ( \\d{4}-[01][0-9]-[0123][0-9])
+    required:
+      - CodiceFiscale
+      - CodiceSoglia
+      - FornituraNucleo
+  ConsultazioneSogliaIndicatoreResponse:
+    type: object
+    properties:
+      Componente:
+        type: array
+        items:
+          $ref: "#/definitions/NucleoType"
+        x-nullable: true
+      TipoIndicatore:
+        type: string
+        enum:
```

Comment:
string is enough (just put the enum into the description of this field)

Reply:
fixed in 0c41302